### PR TITLE
Update Data and formatting.

### PR DESCRIPTION
**Pull request by lilpolymath**
#### Feats
- Updated the data for Lagos state.
- New resolver for each state.

### CODE REVIEW

**Review by mykeels — CHANGES_REQUESTED** [review]
Hey, awesome work @lilpolymath ... Thanks for putting this together so quickly.

- Good one, putting the file names in lowercase. Can all folders be in lower case too? 
- Can all file and folder names with. spaces be hyphenated. E.g. "command-center" instead of "command center".
- The code in the index.js file should be capable of creating all VCF files from emergency-contacts.json, not from a separate lagos.json file
- The lagos.json file is not necessary

**Comment by lilpolymath** [comment]
> Hey, awesome work @lilpolymath ... Thanks for putting this together so quickly.
> 
> * Good one, putting the file names in lowercase. Can all folders be in lower case too?
> * Can all file and folder names with. spaces be hyphenated. E.g. "command-center" instead of "command center".
> * The code in the index.js file should be capable of creating all VCF files from emergency-contacts.json, not from a separate lagos.json file
> * The lagos.json file is not necessary

For the third comment. is there a way we can restructure the data?

File: index.js [hunk 1]
```diff
@@ -1,33 +1,82 @@
...
+    main,
+    area,
+    divisions,
+    commandControl,
+  };
+};
+
+const final = contacts(states);
+
+// console.log(
```
Comment (mykeels) [inline index.js:47]: Can we remove this commented code?

**Comment by mykeels** [comment]
Restructure how?

File: index.js (hunk 1 above)
Comment (lilpolymath) [reply index.js:47]: Yes.

**Comment by lilpolymath** [comment]
Never mind. I got a headway on what I wanted to ask.

**Review by mykeels — APPROVED** [review]

**Pull request merged**